include/oneapi/tbb/detail/_config.h

```diff
@@ -253,7 +253,7 @@
 #define __TBB_CPP17_ALLOCATOR_IS_ALWAYS_EQUAL_PRESENT   (__TBB_LANG >= 201703L)
 #define __TBB_CPP17_IS_SWAPPABLE_PRESENT                (__TBB_LANG >= 201703L)
 
-#define __TBB_RESUMABLE_TASKS                           (!__TBB_WIN8UI_SUPPORT && !__ANDROID__)
+#define __TBB_RESUMABLE_TASKS                           (!__TBB_WIN8UI_SUPPORT && !__ANDROID__ && !__HAIKU__)
```

Comment:
Haiku don't have ucontext.h

Otherwise:
```
/RAMDisk/oneTBB-2021.1.1/src/tbb/co_context.h: In function 'void tbb::detail::r1::create_coroutine(tbb::detail::r1::coroutine_type&, std::size_t, void*)':
/RAMDisk/oneTBB-2021.1.1/src/tbb/co_context.h:175:11: error: 'getcontext' was not declared in this scope
     err = getcontext(&c.my_context);
           ^~~~~~~~~~
compilation terminated due to -Wfatal-errors.
```